jaxrs-tck/pom.xml

```diff
@@ -159,5 +177,18 @@
             <version>3.0.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>5.0.0</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.netbeans.tools</groupId>
```

Comment:
A CQ would be needed for this guy

Reply:
Can one of the committers help create CQ for the same. 

The below jar is used to compile and run the signature test.
```
                <groupId>org.netbeans.tools</groupId>
                <artifactId>sigtest-maven-plugin</artifactId>
                <version>1.4</version>
```

The same jar was used in jakartaee-tck and the CQ raised for it was https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23240. I hope this can be referred to expedite the approval.

@andymc12 @spericas @chkal @spericas

Reply:
All CQs always are valid for *any* EF project, so if a CQ exists for this library opened by another EF project you can simply use this library without any further steps.

Reply:
(This is the benefit of the fact that the EF forces us to use the EF licence)